src/Microsoft.AspNet.Authentication.OAuth/Events/OAuthCreatingTicketContext.cs

```diff
@@ -140,15 +135,15 @@ public class OAuthCreatingTicketContext : BaseContext
         public HttpClient Backchannel { get; }
 
         /// <summary>
-        /// Gets the <see cref="ClaimsPrincipal"/> representing the user.
+        /// The <see cref="AuthenticationTicket"/> that will be created.
         /// </summary>
-        public ClaimsPrincipal Principal { get; set; }
+        public AuthenticationTicket Ticket { get; set; }
```

Comment:
nit: for consistency, consider naming it `AuthenticationTicket` to match the property name exposed by `BaseControlContext` (or rename `BaseControlContext.AuthenticationTicket` to `Ticket` :smile:)